index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>sdf</title>
```

Comment:
I think it would be better to use a more descriptive title. Titles are designed to provide a short piece of text that should stand for the document in cases such as window title bars, bookmark lists, and result lists from search services.

Reply:
This comment is great, you're being polite and describing why it would be good to make this change! Kudos for this one! 💯 🤓